projects/power_select/inc/power_select.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+// Measures values from power selection to send over CAN and handles faults.
+// Requires gpio, interrupts, event queue, CAN, ADC (in ADC_MODE_SINGLE), and soft timers to be
+// initialized.
+
+#include "adc.h"
+#include "can_pack.h"
+#include "can_transmit.h"
+#include "can_unpack.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "soft_timer.h"
+#include "status.h"
+
+#include "power_select_defs.h"
+#include "power_select_events.h"
+#include "power_select_thermistor.h"
+
+#define POWER_SELECT_MEASUREMENT_INTERVAL_MS 1000
+#define POWER_SELECT_MEASUREMENT_INTERVAL_US ((POWER_SELECT_MEASUREMENT_INTERVAL_MS)*1000)
+
+typedef enum {

```

Comment:
Do these enums and `#defines` need to be here? Or can they go into their own `.h`